packages/fxa-admin-server/src/gql/account/account.resolver.ts

```diff
@@ -214,6 +227,7 @@ export class AccountResolver {
       .where('uid', uidBuffer);
   }
 
+  @Features(AdminPanelFeature.ConnectedServices)
```

Comment:
Is the intent here that `ConnectedServices` may require `AccountSearch` permissions plus an additional layer? It looks like they both get `PermissionLevel.Support` for now so I don't immediately see the need for a separate guards, but I guess that's also the case for viewing `LinkedAccounts` so, seems fine either way.

We probably want to [update the `name`](https://github.com/mozilla/fxa/blob/5a44f6e719580430fbeb41097618719024e7edd3/packages/fxa-shared/guards/AdminPanelGuard.ts#L86) from "active sessions" to "connected services" too.

Reply:
I ended up being the one who decided the permission levels for these things as best I could. Although not super likely, I do foresee the possibility that someone requests different permission than what I have assumed, so I've tried to keep these somewhat granular, so they can easily be altered down the road.